tests/performance/density.go

```diff
@@ -85,7 +85,7 @@ var _ = SIGDescribe("Control Plane Performance Density Testing", func() {
 	})
 
 	Describe("Density test", func() {
-		vmCount := 100
+		vmCount := 90
```

Comment:
@acardace @tiraboschi @0xFelix Please revert, this will most likely throw off the performance results...

@rthallisey fyi

Reply:
Why was this changed?  If we were unable to great 100 vms it means we're using more memory and cpu, which we should be reporting in the change log for perf reasons.

Reply:
+ adjust the lane. 
Just for awareness, we see a great increase in the runtime of the tests with this PR. The perf jobs don't show any degradation but that might be because of this change as well.
@tiraboschi would you be able to investigate this and make the revert in parallel?

Reply:
In order to stream out the serial console log we are adding  an additional container executing a sort of tail command.
Since this feature is now enabled by default, the new container is going to be present for all the virt-launcher pods.

If not otherwise customized, the new container is configured with a request of 35M of memory and 5m of CPU:
https://github.com/kubevirt/kubevirt/blob/main/pkg/virt-controller/services/serialconsolelog.go#L68-L75

Having 100 VMs in the performance test lane, this means 3500M more in terms of memory and 500m more in terms of CPU; this was enough to hit the memory boundary of the enviroment and made the test lane failing.
See: https://github.com/kubevirt/kubevirt/pull/10110/files#r1329218961

The 90/100 VMs are create with 100m Request/Limit for CPU and 90Mi for memory (see: https://github.com/kubevirt/kubevirt/blob/main/tests/performance/density.go#L274-L275 ) , adding 35M there just for virt-tail is a considerable increase but that number has been tuned on real usage for real VMs.
If moving back to 100 VMs here is a must, a possible countermeasure is reducing the memory request of virt-tail there just for the purpose of passing the tests, but this is somehow cheating.

Reply:
Imho we should work on getting back to 100 VMs to have an apples-to-apples comparison of the Kubevirt performance.
We just need to adjust the lane to accommodate the increased memory consumption. 
As a follow it is a good idea to have a look if 35M is the bare safe minimum or if we could lower it.

Reply:
Let's try with https://github.com/kubevirt/kubevirt/pull/10571 first reducing the requested memory for the guest-console-log container to 21M (I'm mostly concerned to the limit). If not enough we can only alter the enviroment to fit the lane.